sbt_common/internal_model/src/test/scala/uk/ac/wellcome/models/work/internal/WorkTest.scala

```diff
@@ -59,6 +59,7 @@ class WorkTest extends FunSpec with Matchers with IdentifiersUtil {
     val caught = intercept[JsonDecodingError] {
       fromJson[IdentifiedWork](jsonString).get
     }
-    caught.getMessage shouldBe "Attempt to decode value on failed cursor: DownField(agent),DownArray,DownField(items)"
+    caught.getMessage should startWith(
+      "Attempt to decode value on failed cursor")
```

Comment:
nit: Can you pull this onto a single line?

Reply:
It was. The scala formatter broke it. IMHO the line length limit should be higher...